### DISCUSSION TITLE
Presenting a seminar on the CubeSatSim beta v1.3 on Sat, June 1 at the SEAPAC convention in Seaside OR

### DISCUSSION THREAD

**Discussion by W7DGP** [alanbjohnston/CubeSatSim#316] [General]
Similar to last year, I'm presenting a short discussion and demo of the beta version of the CubeSat. I'm allowing more demo time and only running FSK and BPSK. I'm not sure if I can get the Pi ground station up on the big screen but I'll try. No networking, just RF. I'll have FoxTelem running on the ground station or on my PC. I have a big light for solar panel changes and also demo the accelerometer and gyro. So far the beta has been running well.

**Comment by alanbjohnston** [comment]
Good luck!  Let us know how it goes.

Alan

**Reply by W7DGP** [reply]
The presentation at SEAPAC went very well. I reduced the number of slides shown in order to have more time to run the demo. I ran the FSK mode for the demo and was able to show solar panel, accelerometer and gyro activity. 
I had a group who had a booth at the convention: oresat.org, The Univ of OR based group. They have one sat in orbit and another nearly ready to launch. They didn't know much about AMSAT other than it existed. They expressed high interest in the CubeSatSim project.
In summary, the beta v1.3 ran very well and "orbited" around the room as it was sending telemetry.

**Reply by W7DGP** [reply]
Forgot to include the link to the Oregon group: https://www.oresat.org/satellites
